packages/flutter/lib/fix_data.yaml

```diff
@@ -11,6 +11,98 @@
 version: 1
 transforms:
 
+  # Changes made in https://github.com/flutter/flutter/pull/69805.
```

Comment:
https://github.com/flutter/flutter/pull/68905 ?

Reply:
Hey @pcsosinski what is the question?

Reply:
Oh I see, typo! Thanks!